pkg/server/plugin/notifier/k8sbundle/k8sbundle.go

```diff
@@ -61,6 +61,11 @@ type pluginConfig struct {
 	KubeConfigFilePath string `hcl:"kube_config_file_path"`
 }
 
+type pluginConfig struct {
+	cluster  `hcl:",squash"` // for hcl v2 it should be `hcl:",remain"`
+	Clusters []cluster       `hcl:"clusters"`
+}
+
```

Comment:
Why the need to have two separate structs?

Reply:
I think it has to be this way if we want the cluster config to use the same underlying struct at both the root level and inside the clusters array?

Reply:
💯 The idea is to not repeat the fields while at the same time, provide backward compatibility so old configs won't break. The first struct provides that, while the second one allows to expand into multiple clusters.